package.json

```diff
@@ -20,55 +20,64 @@
     "pretest": "rimraf .coverage",
     "pretest:watch": "npm run pretest",
     "pretest:coverage": "npm run pretest",
+    "size": "size-limit",
     "typecheck": "tsc --noEmit"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/calebdwilliams/adoptedStyleSheets.git"
   },
-  "author": "Caleb D. Williams <caleb.d.williams@gmail.com>",
+  "authors": [
```

Comment:
I don't technically think this is valid in package.json. Can you revert but add a contributors? You certainly deserve credit and I'm more than happy to give it where it's due; really appreciate the effort here.

Reply:
Oh right. I'll replace it with `contributors` instead.